website/content/docs/install.mdx

```diff
@@ -20,8 +20,16 @@ There are several options to install OpenBao:
 
 ## Package manager
 
+### Homebrew - MacOS
+
+```shell-session
+$ brew info openbao
+$ brew install openbao
+```
+
```

Comment:
Not to nitpick too terribly much, but do you mind adding a heading so it is clear the below is for Linux? Thanks!

```suggestion

### Linux

```